cli/src/utils/renderVersionWarning.ts

```diff
@@ -1,6 +1,14 @@
-import { execSync } from "child_process";
 import { getVersion } from "./getT3Version.js";
 import { logger } from "./logger.js";
+import { execSync } from "child_process";
```

Comment:
Looks like its done it again

Reply:
what's wrong here?

Reply:
https://github.com/t3-oss/create-t3-app/pull/1392#discussion_r1186910626
its the line bellow. the comment and import were further down before.

Reply:
oh right